### PR TITLE
Performance enhancements for populating the db.

### PR DESCRIPTION
**Pull request by alanren2**
Made improvements that we can now upload everything in 40 minutes. Ideas for future improvements:
- Use Pypy
- Use multithreading on parse_census_data (need to add logic to first get id_to_county dict filled)
- Perhaps use a mongo function that saves/updates multiple Documents at the same time (ex: insert)

### CODE REVIEW

File: backend/api/views/main.py [hunk 1]
```diff
@@ -97,17 +97,28 @@ def populate_db():
     dates = list(files.values())
     dates.sort()
     date_initial = dates[0]
-    for file, date in files.items():
-        responses = parse_census_data(file, date, date_initial, parse2000)
+    objs = {}
```
Comment (chyku) [inline backend/api/views/main.py]: what's objects contain, can the dict have a more descriptive name or can u comment

**Review by chyku — CHANGES_REQUESTED** [review]
just one thing ty

**Review by chyku — APPROVED** [review]

**Review by caiyingshen — APPROVED** [review]
beep beep

**Comment by alanren2** [comment]
boop boop

**Pull request merged**